Documentation/source/Updating to 4.0.md

```diff
@@ -1,2 +1,49 @@
 Updating to 4.0
 =============================
+
+## What's new?
+---
+
+### Multi-page support
+Documents with multiple pages are now supported, being possible to capture several pages to get the analysis results. To enable it (disabled by default) set the `GiniConfiguration.multipageEnabled` property to `true`.
+Besides, when using the Component API, the `MultipageReviewViewController` component is used to handle one or several documents (see the Example app for implementation details).
+
+### Networking implementation
+The Screen API now offers the option to include all the networking analysis logic, only being necessary to provide a `GiniVisionResultsDelegate` when initializing the Screen API in order to get the analysis results.
+If you want to migrate your Screen API implementation to this one, just remove your current implementation of the `GiniVisionDelegate` and implement the `GiniVisionResultsDelegate`. You have had to add the additional line in your `Podfile` before, as pointed out in the [Integration guide](integration.html).
+
+### Custom photos gallery
+A custom photos gallery picker has been designed, which unlike the native `UIImagePickerController` allows to select multiple photos at the same time, to start caching before showing it (no delays when showing the first time) and some additional customization.
+
+### Gini iOS SDK 1.0
+
+In order to use the new Multipage feature you have to update the [Gini iOS SDK](https://github.com/gini/gini-sdk-ios) to version 1.0, which uses a different way of handling the document analysis, introducing **Partial** and **Composite** documents.
+
+#### Bolts
+In order to use the new Multipage feature, you have to update the [Gini iOS SDK](https://github.com/gini/gini-sdk-ios) to version 1.0 which uses last version of _Bolts_ (1.9). This version introduces a lot of improvements and bug fixes, but also some breaking changes in the syntaxis.
+* `continue()` is now `continueWith(block:)`
+* `continue(successBlock:)` is now `continueOnSuccessWith(block:)`
+* And now every `BFTask` has a specific type for the result, `BFTask<ResultType>`. i.e: `BFTask<GINIDocument>`
+
+
+## Breaking Changes
+---
```

Comment:
Why do you need this horizontal line? Shouldn't it at least be before the header?

Reply:
It shows a line below the header that looks nice.